stamp_sign/static/src/components/sign_request/document_signable.js

```diff
@@ -0,0 +1,31 @@
+import { patch } from "@web/core/utils/patch";
+import { Document } from "@sign/components/sign_request/document_signable";
+
+
+patch(Document.prototype, {
+    getDataFromHTML() {
+        super.getDataFromHTML()
+        const { el: parentEl } = this.props.parent;
+
+        this.company = parentEl.querySelector("#o_sign_signer_company_input_info")?.value;
+        this.address = parentEl.querySelector("#o_sign_signer_address_input_info")?.value;
+        this.city = parentEl.querySelector("#o_sign_signer_city_input_info")?.value;
+        this.country = parentEl.querySelector("#o_sign_signer_country_input_info")?.value;
+        this.vat = parentEl.querySelector("#o_sign_signer_vat_input_info")?.value;
+        this.logo = parentEl.querySelector("#o_sign_signer_logo_input_info")?.value;
```

Comment:
You can store it in one variable. And do it with loop (key, value).so it is easy to scalable.